tests/lib/snaps/store/test-snapd-load-generator/load-generator

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+-d#!/usr/bin/env python3
```

Comment:
I've never seen the `-d` prefix to a shebang, is this a typo or does it do something intentional?